### PR TITLE
Blog app integration

### PR DESCRIPTION
**Pull request by rbreva**
### Blog app - integration
<hr>

:ballot_box_with_check: I have read this repository's contribution guidelines.
:ballot_box_with_check: My pull request has a descriptive title (not a vague title like Update index.MD)
:ballot_box_with_check: My pull request targets the main branch of this [project](https://github.com/rbreva/BlogApp).
:ballot_box_with_check: I added relevant documentation for all the files I changed

<hr>

-This PR solves the N + 1This PR solves the N + 1 problem and writes some integration tests
Captura01
Captura02

- Write user tests
- Write post tests

<hr>

👤 **Mauricio Mendoza**

- GitHub: [@rbreva](https://github.com/rbreva)
- Twitter: [@rbreva75](https://twitter.com/rbreva75)
- LinkedIn: [LinkedIn](https://www.linkedin.com/in/r-mauricio-mendoza-huerta/)

👤 **Didier Ganthier**

- GitHub: [@didierganthier](https://github.com/didierganthier)
- Twitter: [@didierganthier_](https://twitter.com/didierganthier_)
- LinkedIn: [Didier Peran Ganthier](https://linkedin.com/in/didierganthier)

<hr>

### CODE REVIEW

File: spec/integration/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    it "renders the user's bio" do
+      expect(page).to have_content(@user.bio)
+    end
+
+    it "renders the user's last 3 posts" do
+      expect(page).to have_content(@first_post.title)
+      expect(page).to have_content(@second_post.title)
+      expect(page).to have_content(@third_post.title)
+    end
+  end
```
Comment (Whoistolu) [inline spec/integration/user_spec.rb]: - [ ] [REQUIRED] Kindly note that for the featured tests for the User show page: You have missed the last 3 tests:
* Tests that assert you can see a button that lets me view all of a user's posts.
* Tests that redirects to the post's show page when you click on a user's post.
* Tests that redirect to the user's post's index page when you click on a **see all posts**.

Kindly add these tests before asking for another review.

File: spec/integration/post_spec.rb [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      expect(page).to have_content(@fourth_post.likes_counter)
+      expect(page).to have_content(@second_post.likes_counter)
+      expect(page).to have_content(@third_post.likes_counter)
+    end
+
+    it "redirects to a specific post's show page" do
+      click_link @second_post.title
+      expect(page).to have_current_path(user_post_path(@user, @second_post))
+    end
+  end
```
Comment (Whoistolu) [inline spec/integration/post_spec.rb]: - [ ] [REQUIRED] Please take a look at the featured tests for `User` **post index page**: You have missed out on this test:
- Tests that assert you can see a section for pagination if there are more posts than fit on the view.

Kindly fix this before asking for another review.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @rbreva 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._
- [ ] Great job licensing this project but let's update the License year to 2023 as this is the year you built this project. Great job 👍🏽 

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -121,6 +121,13 @@ rails server
...
+👤 **Didier Ganthier**
+
+- GitHub: [@didierganthier](https://github.com/didierganthier)
+- Twitter: [@didierganthier_](https://twitter.com/didierganthier_)
+- LinkedIn: [Didier Peran Ganthier](https://linkedin.com/in/didierganthier)
+
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🔭 Future Features <a name="future-features"></a>
```
Comment (FrederickMih) [inline README.md:133]: - As pointed out by the previous reviewer, your license file still contains obsolete information as you can see in the screenshot below:

Capture

- You really need to update the required information(date and name)

**Review by FrederickMih — APPROVED** [review]
## Hi 🙋 @rbreva  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 

- PR is professional ✔️ 
- All tests scenarios covered ✔️ 
- Linter checks passed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**